src/OTSession.js

```diff
@@ -77,18 +90,42 @@ export default class OTSession extends Component {
   getSessionInfo() {
     return this.state.sessionInfo;
   }
+
+  onStreamCreated = event => {
+    const index = this.state.streams.findIndex(stream => stream.id === event.streamId);
+    if (index < 0) {
+      this.setState({
+        streams: [...this.state.streams, event.streamId]
+      })
+    }
+  }
+
+  onStreamDestroyed = event => {
```

Comment:
I think each individual component should be handling the streamCreated/streamDestroyed events that way it can be flexible b/c the `OTPublisher` component will also need to fix this.